o1vm/src/pickles/prover.rs

```diff
@@ -328,6 +480,55 @@ where
             .collect(),
     };
 
+    let logup_evaluations = lookup_env.as_ref().map(|lookup_env| LookupProof {
```

Comment:
sorry for nitpicking, but I'd like if we use only one term amongst lookup and logup
Or have a clear convention as which term refers to what

Reply:
Chose `lookup`